workflow-templates/sphinx.yml

```diff
@@ -0,0 +1,40 @@
+# Builds the Sphinx documentation in the specified folder.
+# Default values are docs/ for the directory where the Sphinx
+# documentation is located and true to run linkcheck
+# To pass a new folder, set directory_path to the folder path
+# To disable linkcheck, set check_link to any value 
+
+---
+name: sphinx
+
+on: 
+  push:
+  pull_request:
+  schedule:
+    - cron: '0 0 * * 0'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    env: 
+      directory_path:
+      check_link: true # set to false to remove linkcheck
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+      - name: Sphinx install
+        run: pip install --upgrade sphinx
```

Comment:
This may bring some gotchas (It's happened before) but I assume we will notice when it happens.